eq-author-api/db/index.js

```diff
@@ -1,4 +1,48 @@
-var environment = process.env.NODE_ENV || "development";
-var config = require("../config/knexfile.js")[environment];
+const config = require("../config/knexfile.js");
+const createLogger = require("../utils/createLogger");
+const pinoMiddleware = require("express-pino-logger");
+const pino = pinoMiddleware();
+const logger = createLogger(pino.logger);
 
-module.exports = require("knex")(config);
\ No newline at end of file
+const getConfig = async function(secretId) {
```

Comment:
This call is made a lot now and is now making a network call every time.

Could we do this once at startup?